templates/pgbouncer/pgbouncer-networkpolicy.yaml

```
@@ -1,7 +1,7 @@
 ################################
 ## Pgbouncer NetworkPolicy
 #################################
-{{- if .Values.networkPolicies.enabled }}
+{{- if and .Values.networkPolicies.enabled .Values.pgbouncer.networkPolicies.enabled }}
```

We should probably have `.Values.pgbouncer.enabled` too, right?

```suggestion
{{- if and  .Values.pgbouncer.enabled .Values.networkPolicies.enabled .Values.pgbouncer.networkPolicies.enabled }}
```